spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/seek.adoc

```diff
@@ -184,6 +184,9 @@ public class SeekToLastOnIdleListener extends AbstractConsumerSeekAware {
 
     /**
     * Rewind all partitions one record.
+    * As of version 3.3, for multi-group listeners, it's recommended to use `getTopicsAndCallbacks()`
```

Comment:
There is no reason to mention this in the docs since the version of the doc is exact match to the code.
So, better to just adjust these samples in the doc to whatever is expected from the code.